modules/installation-restricted-network-samples.adoc

```diff
@@ -15,23 +15,20 @@ endif::[]
 [id="installation-restricted-network-samples_{context}"]
 = Using Cluster Samples Operator image streams with alternate or mirrored registries
 
-Most image streams in the `openshift` namespace managed by the Cluster Samples Operator
-point to images located in the Red Hat registry at link:https://registry.redhat.io[registry.redhat.io].
+[role="_abstract"]
+Most image streams in the `openshift` namespace managed by the Cluster Samples Operator point to images located in the Red Hat registry at link:https://registry.redhat.io[registry.redhat.io].
```

Comment:
This lead-in sentence ideally should be framed around what a user can do. This is where things get challenging as we should also not repeat the title text in the lead-in sentence.